device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py

```diff
@@ -1,25 +1,60 @@
 #!/usr/bin/env python
+#
+# Platform-specific SFP transceiver interface for SONiC
+#
 
 try:
-    from sonic_sfp.sfputilbase import sfputilbase
-except ImportError, e:
-    raise ImportError (str(e) + "- required module not found")
+    import time
+    from sonic_sfp.sfputilbase import SfpUtilBase
+except ImportError as e:
+    raise ImportError("%s - required module not found" % str(e))
 
 
-class sfputil(sfputilbase):
-    """Platform specific sfputil class"""
+class SfpUtil(SfpUtilBase):
+    """Platform-specific SfpUtil class"""
 
-    port_start = 0
-    port_end = 31
-    ports_in_block = 32
+    PORT_START = 0
+    PORT_END = 31
+    PORTS_IN_BLOCK = 32
 
-    port_to_eeprom_mapping = {}
+#    EEPROM_OFFSET = 20
```

Comment:
no need this line?

Reply:
Right, forgot to remove it, will send another patch right away.